tools/ctu-ci/analyze_project.sh

```diff
@@ -46,76 +46,76 @@ do
-#NOXTU-RUN
-#../../../clang_build/clang/tools/xtu-build-new/xtu-analyze.py $MEMPROF --record-coverage -b passed_buildlog.json -o .xtu-out-noxtu -j 16 -v --clang-path /mnt/storage/xtu-service/clang_build/buildrelwdeb/bin/ --analyze-cc-path /mnt/storage/xtu-service/clang_build/clang/tools/scan-build-py/bin/ --no-xtu | tee "$PROJNAME"_noXTU.out
-PATH=/mnt/storage/xtu-service/clang_build/buildrelwdeb/bin/:$PATH $CC analyze -o .xtu-out-noxtu --analyzers clangsa -j 16 buildlog.json | tee "$PROJNAME"_noXTU.out
-#mkdir "$PROJNAME"_gcovNoXtu
-#gcovr -k -g .xtu-out-noxtu/gcov --html --html-details -r . -o "$PROJNAME"_gcovNoXtu/coverage.html
-#XTU RUN (on files that dont crash)
-#../../../clang_build/clang/tools/xtu-build-new/xtu-analyze.py $USR $MEMPROF $REPARSE --record-coverage -b passed_buildlog.json -o .xtu-out-xtu -j 16 -v --clang-path /mnt/storage/xtu-service/clang_build/buildrelwdeb/bin/ --analyze-cc-path /mnt/storage/xtu-service/clang_build/clang/tools/scan-build-py/bin/ | tee "$PROJNAME"_XTU.out
-PATH=/mnt/storage/xtu-service/clang_build/buildrelwdeb/bin/:$PATH $CC analyze -o .xtu-out-xtu --analyzers clangsa --ctu-collect $REPARSE $STRICT_MODE -j 16 buildlog.json | tee "$PROJNAME"_XTU_build.out
-PATH=/mnt/storage/xtu-service/clang_build/buildrelwdeb/bin/:$PATH $CC analyze -o .xtu-out-xtu --analyzers clangsa --ctu-analyze $REPARSE $STRICT_MODE -j 16 buildlog.json | tee "$PROJNAME"_XTU_analyze.out
+rm -rf .ctu/
+rm -rf .ctu-out-init/
+rm -rf .ctu-out-noctu/
+cp -rf .ctu-out-ctu ./"$PROJNAME"_ctu-out-ctu
+rm -rf .ctu-out-ctu/
+#INITIAL CTU RUN
+#/usr/bin/time -f "%e" ../../../clang_build/clang/tools/ctu-build-new/ctu-build.py $USR $REPARSE -b buildlog.json --clang-path /mnt/storage/ctu-service/clang_build/buildrelwdeb/bin/ -v -j 16 2>"$PROJNAME"_buildtime.out | tee "$PROJNAME"_build.out
+#../../../clang_build/clang/tools/ctu-build-new/ctu-analyze.py $USR $REPARSE -b buildlog.json -o .ctu-out-init -j 16 -v --clang-path /mnt/storage/ctu-service/clang_build/buildrelwdeb/bin/ --analyze-cc-path /mnt/storage/ctu-service/clang_build/clang/tools/scan-build-py/bin/ --log-passed-build passed_buildlog.json | tee "$PROJNAME"_init.out
+#NOCTU-RUN
+#../../../clang_build/clang/tools/ctu-build-new/ctu-analyze.py $MEMPROF --record-coverage -b passed_buildlog.json -o .ctu-out-noctu -j 16 -v --clang-path /mnt/storage/ctu-service/clang_build/buildrelwdeb/bin/ --analyze-cc-path /mnt/storage/ctu-service/clang_build/clang/tools/scan-build-py/bin/ --no-ctu | tee "$PROJNAME"_noCTU.out
+PATH=/mnt/storage/ctu-service/clang_build/buildrelwdeb/bin/:$PATH $CC analyze -o .ctu-out-noctu --analyzers clangsa -j 16 buildlog.json | tee "$PROJNAME"_noCTU.out
```

Comment:
Some of these absolute paths are environment specific, and should not be renamed.